subprojects/core/src/main/java/org/gradle/initialization/buildsrc/BuildSrcBuildListenerFactory.java

```diff
@@ -32,17 +32,12 @@
 
     private static final String DEFAULT_BUILD_SOURCE_SCRIPT_RESOURCE = "defaultBuildSourceScript.txt";
 
-    Listener create(boolean rebuild) {
-        return new Listener(rebuild);
+    Listener create() {
+        return new Listener();
```

Comment:
I am not sure I quite understand this point. We are deleting the `buildDir` directory. That's what the `clean` task does. Are you saying that there could be some logic that hooks into the `clean` task which now wouldn't happen anymore?